src/main/java/org/terasology/metalrenegades/world/dynamic/HumidityProvider.java

```diff
@@ -65,7 +67,7 @@ public void process(GeneratingRegion region) {
         // TODO: Setup humidity
```

Comment:
Remove TODO comments like this, since the humidity/temperature is now set up by this PR.